cdap-docs/application-templates/source/etl/templates/validators/core.rst

```diff
@@ -0,0 +1,59 @@
+.. meta::
+:author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+================
+Validators: Core
+================
+
+.. rubric:: Description
+
+A System Validator that offers a set of function's that can be called from validator transform.
```

Comment:
A system-supplied validator that offers a set of functions that can be called from a validator transform.